app/views/candidates/feedbacks/show.html.erb

```diff
@@ -4,14 +4,14 @@
   <div class="govuk-grid-column-two-thirds">
     <div class="govuk-panel govuk-panel--confirmation">
       <h1 class="govuk-panel__title">
-        Thank you for your feedback.
+        Thank you for your feedback
       </h1>
     </div>
-    <h2 class="govuk-heading-m">
-      View and request more school experience
-    </h2>
-    <p>
-      <%= link_to 'Your requests and bookings', candidates_dashboard_path %>
+   <h2 class="govuk-heading-m">
```

Comment:
Preview:

<img width="518" alt="image" src="https://github.com/DFE-Digital/schools-experience/assets/35870975/651fe0b4-3210-4921-ad88-8edcd97a7e0f">

Reply:
This is a screen provided when a candidate gives feedback through this link in the footer: 
![image](https://github.com/DFE-Digital/schools-experience/assets/35870975/3a813a4e-21bf-457b-b83a-6ca02038eb47)